scanners/httpobs_scanner.py

```diff
@@ -0,0 +1,34 @@
+import requests
+import time
+
+
+class HTTPObservatoryScanner():
+    ## TODO: Re-write this with the httpobs python module
```

Comment:
That makes sense, to keep our code as minimal as possible, but using this as a wrapper still makes sense for consistency.

Reply:
Actually, I will leave this as is for now, as it's working very well with the queue implementation.
Might change it in the future.

Reply:
I'm cool with that.